source/_integrations/saj.markdown

```diff
@@ -12,36 +12,62 @@ The `saj` sensor will poll a [SAJ](https://www.saj-electric.com/) solar inverter
 
 This sensor uses the web interface and to use it, you have to be able to connect to the solar inverter from your favorite web browser.
 
+There is a difference between inverters that are connected via an ethernet module and those connected via a WiFi module.
+The WiFi module requires a username and password for authentication where the ethernet module does not.
+
 ## Configuration
 
 To enable this sensor, add the following lines to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
+# Example configuration.yaml entry for ethernet module
+sensor:
+  - platform: saj
+    host: IP_ADDRESS_OF_DEVICE
+```
+
+```yaml
+# Example configuration.yaml entry for WiFi module
 sensor:
   - platform: saj
     host: IP_ADDRESS_OF_DEVICE
+    type: wifi
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Now if a user has an inverter connected via ethernet cable he can indeed just copy-&-paste the sample config (without type, username, password) and it will work.
But if a user has an inverter connected via WiFi it will not work with the sample config and he needs to specify at least the type "wifi".

Just asking for your opinion, wouldn't it be better that the "type" parameter is required then (breaking change) ?

Reply:
That is a code decision and not related directly to the documentation.
The documentation is here to reflect the state of the code.

I think it would wise to discuss this on the code base PR.